src/posthog-persistence.ts

```diff
@@ -35,6 +36,7 @@ export const RESERVED_PROPERTIES = [
     SESSION_ID,
     ENABLED_FEATURE_FLAGS,
     USER_STATE,
+    POSTHOG_QUOTA_LIMITED,
```

Comment:
Not sure if we want to persist this tbh.
There are too many things that can change around quota limits (see my comments on the other PR)
Given that this code _drops_ data I would rather be _less_ droppy than more.

I think it's totally fine to have the SDK load, try and send an event and if it fails, pause ingesting events for a fixed period of time (something small like 5-10 mins). Given that this is more to help relieve some load on us I think its best we keep it simple and minimal.

Reply:
xhr is stateless at the mo' so if we're going to pause sending for X minutes it has to store somewhere